Lushu/src/main/kotlin/lushu/ContextGrammar/Grammar/Node.kt

```diff
@@ -0,0 +1,75 @@
+package lushu.ContextGrammar.Grammar
+
+class Node(
+    private val regex: String = "",
+    private val sensitive: Boolean = false,
+    private val star: Boolean = false,
+    private val terminal: Boolean = false,
+    private val children: MutableList<Node> = mutableListOf(),
+) {
+
+    fun getRegex(): String {
+        return this.regex
+    }
+
+    fun getChildren(): MutableList<Node> {
```

Comment:
```suggestion
    fun getChildren(): List<Node> {
```
There's no need to ever have `MutableList` as a return type